docs/source/create_channel/create_channel_config.md

```diff
@@ -0,0 +1,296 @@
+# Using configtx.yaml to build a channel configuration
+
+A channel is created by building a channel creation transaction artifact that specifies the initial configuration of the channel. The **channel configuration** is stored on the ledger, and governs all the subsequent blocks that are added to the channel. The channel configuration specifies which organizations are channel members, the ordering nodes that can add new blocks on the channel, as well as the policies that govern channel updates. The initial channel configuration, stored in the channel genesis block, can be updated through channel configuration updates. If a sufficient number of organizations approve a channel update, a new channel config block will govern the channel after it is committed to the channel.
+
+While it is possible to build the channel creation transaction file manually, it is easier to create a channel by using the `configtx.yaml` file and the [configtxgen](../commands/configtxgen.html) tool. The `configtx.yaml` file contains the information that is required to build the channel configuration in a format that can be easily read and edited by humans. The `configtxgen` tool reads the information in the `configtx.yaml` file and writes it to the [protobuf format](https://developers.google.com/protocol-buffers) that can be read by Fabric.
+
+You can use this tutorial to learn how to use the `configtx.yaml` file to build the initial channel configuration that is stored in the genesis block. The tutorial will discuss the portion of channel configuration that is built by each section of file. Because different sections of the file work together to create the policies that govern the channel, we will discuss channel policies in a [their own section](#channel-policies).
```

Comment:
After reading the entire topic I came back to this section. I think it deserves more focus. Maybe a heading like **Purpose of this tutorial** or something like that.

I love how you take apart and explain each element in each section - very well done. I was not prepared for that  when I started reading this topic. Would it benefit from links here to each section below? I'm torn because I think it is useful to read from top to bottom to understand it all and more importantly how the Profiles section uses the content from the other sections. But when I come back later and want to re-read a section it might be nice to have a link directly to the portion of the file that I am interested in. Maybe it will already be there in the left toc? Certainly not a requirement but a nice to have.

Reply:
I will put an overview heading after the first two paragraphs to break up the text. A list of sections of links wont either, and will help tie the channel policies together as a section.